### DISCUSSION TITLE
Release 2.0.24 for the new jetty CVE

### DISCUSSION THREAD

**Discussion by natan-abolafya** [dropwizard/dropwizard#4792] [General]
Hello.

Not sure if this is a good platform to ask for a release but could you release 2.0.24 so we can avoid the [new jetty vulnerability](https://nvd.nist.gov/vuln/detail/CVE-2021-34429) that is fixed at 9.4.43?

For a little more context: Our builds (as I suspect many others) fail when a vulnerability is found which is a blocking issue. We can ignore specific vulnerabilities of course, which we do when there is no other way, but would rather not do that.

**Comment by joschi** [comment]
@natan-abolafya Thanks for bringing this up!

We've just released Dropwizard 2.0.24 with the latest version of Jetty 9.4.x:
https://github.com/dropwizard/dropwizard/releases/tag/v2.0.24

The artifacts should be available on Maven Central within a few hours.

**Comment by natan-abolafya** [comment]
Thank you @joschi
